### PR TITLE
SureXeroLite (Independent Publisher)

### PR DESCRIPTION
**Pull request by VadimThe848Group**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by VadimThe848Group** [comment]
@microsoft-github-policy-service agree [company="The 848 Group"]

**Comment by VadimThe848Group** [comment]
@microsoft-github-policy-service agree

**Comment by vmanoharas** [comment]
Hello @VadimThe848Group, 

Thank you for submitting your proposal. Please let us know when your connector is prepared for review.

**Comment by VadimThe848Group** [comment]
I have added the connector swagger and properties files. Thanks

**Comment by vmanoharas** [comment]
Hello @VadimThe848Group 
Please remove Proposal word from the heading of this PR or close this PR and create a new PR.

**Review by vmanoharas — COMMENTED** [review]
Hello @VadimThe848Group
Please remove Proposal word from the heading of this PR or close this PR and create a new PR.

**Comment by VadimThe848Group** [comment]
I have renamed the PR. Thanks.

File: independent-publisher-connectors/SureXeroLite/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# SureXeroLite (The 848 Group)
+
```
Comment (vmanoharas) [inline independent-publisher-connectors/SureXeroLite/README.md:2]: Kindly refer the below link and update the readme.md file, mainly please add the required fields.

**Review by vmanoharas — COMMENTED** [review]
Hello @VadimThe848Group, 

thank you for resolving previous comments.  Please resolve other comment.

File: independent-publisher-connectors/SureXeroLite/README.md (hunk 1 above)
Comment (VadimThe848Group) [reply independent-publisher-connectors/SureXeroLite/README.md:2]: Thanks for your comment. I don't think I understand that. Where is the link? and what fields I need to add? Thanks Vadim

Comment (vmanoharas) [reply independent-publisher-connectors/SureXeroLite/README.md:2]: It's my bad forgot to paste the link. 
here you go. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md

Comment (VadimThe848Group) [reply independent-publisher-connectors/SureXeroLite/README.md:2]: I have updated the readme.md file/

**Comment by VadimThe848Group** [comment]
image
image
image
image

**Comment by VadimThe848Group** [comment]
image

**Review by vmanoharas — APPROVED** [review]
Hello @VadimThe848Group, 
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by vmanoharas** [comment]
[[certify-connector]]

**Comment by mallea848** [comment]
@vmanoharas is there any estimate of when we might see this moved to production and become available via the connectors page in Power Platform?
@VadimThe848Group fyi

**Comment by vmanoharas** [comment]
Hi @VadimThe848Group, 

Please accept my sincere apology, due to Thanks giving, Christmas and New Year holidays, there is a deployment freeze until Jan 12th.  We will publish this connector after Jan 12th, from then it will take 4 to 5 weeks.

**Comment by mallea848** [comment]
> Hi @VadimThe848Group,
> 
> Please accept my sincere apology, due to Thanks giving, Christmas and New Year holidays, there is a deployment freeze until Jan 12th. We will publish this connector after Jan 12th, from then it will take 4 to 5 weeks.

thanks for the update

**Comment by vmanoharas** [comment]
Hello @VadimThe848Group, 

Your connector will be considered for production deployment from next week.  Please allow us 3 to 4 weeks to deploy all the regions.

**Comment by mallea848** [comment]
> Hello @VadimThe848Group,
> 
> Your connector will be considered for production deployment from next week. Please allow us 3 to 4 weeks to deploy all the regions.

@vmanoharas cannot see the connector as available in the UK region which is our primary focus. Please can you update us on whether it is available anywhere and when it will be available in the UK?

**Comment by vmanoharas** [comment]
Hello @VadimThe848Group, 

Now the connector deployed on Asia region, it would take a week to deploy to UK region.  

Thanks for your patience and understanding.

**Comment by mallea848** [comment]
> Hello @VadimThe848Group,
> 
> Now the connector deployed on Asia region, it would take a week to deploy to UK region.
> 
> Thanks for your patience and understanding.

thanks

**Comment by vmanoharas** [comment]
Hello Partner,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.

**Comment by mallea848** [comment]
Great news on the release to the UK. Thanks for organising. @vmanoharas

**Comment by mallea848** [comment]
@vmanoharas I have noticed that the publisher says Vadim Khoruzhenko but should read The 848 Group [see link below]. How do we resolve this?

[SureXeroLite on Learn](https://learn.microsoft.com/en-us/connectors/surexerolite/)

**Comment by vmanoharas** [comment]
Hello @mallea848, 

Please see the screenshot below, the documentation is ready as per the information provided in the apiProperties.json file. If you would like to change this, please submit update of your connector.


image

**Pull request merged**